packages/components/organisms/f-checkout/src/store/checkout.module.js

```diff
@@ -159,23 +159,26 @@ export default {
          * @param {Object} payload - Parameter with the different configurations for the request.
          */
         // eslint-disable-next-line no-unused-vars
-        updateCheckout: async ({ commit, state, dispatch }, {
-            url, data, timeout
-        }) => {
+        updateCheckout: async ({
+            commit, state, dispatch, rootGetters
+        }, {
+                url, data, timeout
```

Comment:
Nit: You can probably combine this line with the lines above and below

Reply:
Prettier did this 🙁

Reply:
Ok that's fine, I can agree to disagree with Prettier